feature-libs/cart/base/core/http-interceptors/handlers/bad-cost-center-request.handler.ts

```diff
@@ -0,0 +1,55 @@
+import { Injectable } from "@angular/core";
+import { HttpErrorResponse, HttpRequest } from "@angular/common/http";
+import { isEntityValidationError} from "../../utils/utils";
+import {
+  ErrorModel,
+  GlobalMessageType,
+  HttpErrorHandler,
+  HttpResponseStatus,
+  Priority,
+} from '@spartacus/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class BadCostCenterRequestHandler extends HttpErrorHandler {
```

Comment:
It seems that this error handler should be moved to the b2b checkout feature.  The endpoint to set the cost center on the cart, which produces the error we want to handle is part of the b2b checkout feature and is called within there.